src/main/java/org/zalando/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -177,6 +177,7 @@ public boolean partitionExists(final String topic, final String partition) throw
 
             producer.send(kafkaRecord, ((metadata, exception) -> {
                 if (null != exception) {
+                    LOG.warn("Failed to publish to kafka topic " + topicId, exception);
```

Comment:
I thought that error is really error. 
Now it may be just information about leader change and so on... For now it is just warning